supabase/run_migration.py

```
@@ -0,0 +1,72 @@
+            except Exception as stmt_error:
+                print(f"Error executing statement: {str(stmt_error)}")
+                return 1
+        else:
+            # Split the SQL into individual statements
+            statements = sql.split(';')
+            
+            for statement in statements:
+                # Skip empty statements
+                if statement.strip():
+                    try:
+                        result = client.postgrest.rpc('exec_sql', {'sql_string': statement}).execute()
```

Ruff (0.13.1)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)